eclass/distutils-r1.eclass

```diff
@@ -628,22 +638,14 @@ distutils_enable_tests() {
 	_DISTUTILS_TEST_RUNNER=${1}
 	python_test() { distutils-r1_python_test; }
 
-	local test_deps=${RDEPEND}
-	if [[ -n ${test_pkg} ]]; then
+	if [[ -n ${test_pkgs} ]]; then
 		if [[ ! ${DISTUTILS_SINGLE_IMPL} ]]; then
-			test_deps+=" ${test_pkg}[${PYTHON_USEDEP}]"
+			test_deps+=" ${test_pkgs//'${PYTHON_USEDEP}'/${PYTHON_USEDEP}}"
 		else
 			test_deps+=" $(python_gen_cond_dep "
-				${test_pkg}[\${PYTHON_USEDEP}]
+				${test_pkgs}
 			")"
 		fi
```

Comment:
I'll be honest, this is a hard place for me to verify correctness. I would just verify just incase by converting an example package of each variant, and then use `pquery --max -v --attr raw_bdepend ${PKG}` to verify the result.

Reply:
Yeah. Except I just did `die "${BDEPEND}"` xD.